### PR TITLE
Expand verified option

### PR DESCRIPTION
**Pull request by christianbean**
- an option on `auto_search` to not expand verified classes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - When expanding a class with a strategy, you can now create rules where the
   parent is not the class passed to the strategy.
 - The 'get_equations' method now handles multiple parameters
+- an option on `auto_search` to not expand verified classes
```
Comment (jaypantone) [inline CHANGELOG.md:14]: Is this meant to be here, or up in [Unreleased]?

File: comb_spec_searcher/specification.py [hunk 2]
```diff
@@ -47,6 +47,9 @@ class CombinatorialSpecification(
     A combinatorial specification is a set rules of the form a -> b1, ..., bk
     where each of the bi appear exactly once on the left hand side of some
     rule.
+
+    The defualt is to expand verified classes, but this can be turned off by
```
Comment (jaypantone) [inline comb_spec_searcher/specification.py:51]: defualt -> default

File: CHANGELOG.md (hunk 1 above)
Comment (christianbean) [reply CHANGELOG.md]: fixed

File: comb_spec_searcher/specification.py (hunk 2 above)
Comment (christianbean) [reply comb_spec_searcher/specification.py]: fixed

**Pull request merged**